src/microbe_stage/Microbe.Contact.cs

```diff
@@ -611,13 +611,21 @@ public void SendOrganellePositionsToMembrane()
 
         var organellePositions = new List<Vector2>();
 
+        // This is a workaround for https://github.com/Revolutionary-Games/Thrive/issues/4117
```

Comment:
This also needs to be an XML comment.

Reply:
The format checks didn't allow an XML comment for a local variable, so I left this as is.

Reply:
Oh sorry, I didn't realize that this was not a field. I thought for sure that this was a field considering the vector above `var organellePositions = new List<Vector2>();` But yeah, I see now, this needs to be basically like this.